Add select parameter in GET /contacts (#222)
mautic/developer-documentation-new: docs/rest_api/contacts.rst

     - An array of advanced where conditions
   * - ``order``
     - An array of advanced order statements
   * - ``select``

>      - An array of fields to include in the lead data, ID is required. If not provided, all fields are returned.

Reviewer: I'd say "to field aliases" to be sure to properly understand.

>      - An array of field aliases to include in the lead data, ID is required. If not provided, all fields are returned.

Reviewer 2: ```suggestion
     - An array of field aliases to include in the Contact data, with ID a mandatory field. Mautic returns all fields if it's not provided.
```

I think this should be Contact, or are we referring here to some legacy code where it still refers to lead? Also fixed the Vale warning about passive voice.

Author: Thank you for your response, it has now been updated.
